tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -935,6 +916,25 @@ async function checkInClusterEventDeliveryHelper(targetNamespace, encoding) {
   return ensureInClusterEventReceivedWithRetry(mockHost, eventId);
 }
 
+async function checkInClusterLegacyEvent(targetNamespace) {
+  const eventId = getRandomEventId("legacy");
+  const mockHost = await getVirtualServiceHost(targetNamespace, 'lastorder');
+
+  if (isDebugEnabled()) {
+    await printStatusOfInClusterEventingInfrastructure(targetNamespace, "legacy", 'lastorder');
+  }
+
+  const eventData = {
+    'event-type': 'order.received',
+    'event-type-version': 'v1',
+    'event-time': '2020-09-28T14:47:16.491Z',
+    'data': {'id': eventId, 'legacyOrder': '987'},
+  };
```

Comment:
For consistency, I highly recommend to move constructing the event payload inside `kyma/tests/fast-integration/test/fixtures/commerce-mock/lastorder-function.yaml` here: `function getEventPayloadAndHeaders`.

It can be done in a later ticket to not block this PR.

Reply:
I will do it in a separate PR, to keep this PR unblocked.